apps/web/src/components/Shared/Navbar/BottomNavigation.tsx

```diff
@@ -14,7 +14,7 @@ const BottomNavigation = () => {
   const isActivePath = (path: string) => router.pathname === path;
 
   return (
-    <div className="fixed pb-safe inset-x-0 bg-white border-t border-gray-200 dark:border-gray-800 dark:bg-black bottom-0 z-10 md:hidden">
+    <div className="fixed pb-safe inset-x-0 bg-white border-t border-gray-200 dark:border-gray-800 dark:bg-black bottom-0 z-[5] md:hidden">
```

Comment:
**question:** why arbitrary value?

Reply:
Its okay to have stack index of 5 or less than 10 for bottom bar, so that most of our modals have z-10, causes no problem in overflows.